packages/core/src/components/dropdown-button/dropdown-button.scss

```
@@ -8,41 +8,66 @@
  */
 @use 'misc/common-variables' as vars;
 @use 'mixins/shadow-dom/component';
+@use './dropdown-button.vars' as *;
 
 @mixin triangle($name: '') {
   @if ($name != '') {
```

Stylelint (17.14.1)

[error] Unexpected () used to surround statements for `@-rules` (scss/at-rule-conditional-no-parentheses)

(scss/at-rule-conditional-no-parentheses)